core/domain/param_domain.py

```diff
@@ -75,9 +70,6 @@ def from_dict(cls, param_spec_dict):
     def validate(self):
         """Validate the existence of the object class."""
 
-        # Ensure that this object class exists.
-        object_registry.Registry.get_object_class_by_type(self.obj_type)
-
```

Comment:
Can we still somehow check this?

Reply:
Can we maybe somehow modify the code so that we can check if `self.obj_type` belongs to some constant (we probably need to create it) and then write a backend test that will verify that the constant is up-to-date with the `object_registry`. That way we can just use `feconf` here, but the self.obj_type` will still stay the same.`

Reply:
Yeah, we can go for the constant but one thing I am not sure about verifying whether the constant is up to date or not because we have only one supported type in `ParamSpec` class and that is `UnicodeString`, and our constant will have only this type. So, rather can we check the values in the constant whether they are valid or not? To show the changes, I have pushed them in this [commit](https://github.com/oppia/oppia/pull/15048/commits/7679b8a4fc15de674be2593ce6b6f2f06335be21). please have a look and give me your opinion.